test/validation/api/ipsec/ipsec.c

```diff
@@ -139,6 +139,15 @@ int ipsec_check(odp_bool_t ah,
 	     ODP_SUPPORT_NO == capa.op_mode_inline_out))
 		return ODP_TEST_INACTIVE;
 
+	if (suite_context.queue != ODP_QUEUE_INVALID) {
+		if (suite_context.q_type == ODP_QUEUE_TYPE_PLAIN &&
+		    !capa.queue_type_plain)
+			return ODP_TEST_INACTIVE;
```

Comment:
This accidentally inactivates all tests for an implementation that supports only sync mode and does not support queues of any type.